server/app/services/rpc/container_handler.rb

```diff
@@ -14,9 +13,10 @@ def initialize(grid)
       @grid = grid
       @logs = []
       @stats = []
-      @cached_container = nil
-      @logs_buffer_size = 5
+      @cached_containers = {}
+      @logs_buffer_size = 10
```

Comment:
Because it holds on to the buffered logs indefinitely, this change to increase the logs buffer size can break things like the https://github.com/kontena/kontena/pull/1874#issuecomment-284659342 e2e specs, where there's nothing else getting logged :(

IMO if you want to increase the logs buffering any more, then it needs to also flush them out after e.g. 1s, or this will start to behave too much like a bug from the perspective of users..